lib/ansible/modules/clustering/k8s/k8s.py

```diff
@@ -99,6 +99,14 @@
       the generated hash and append_hash=no)
     type: bool
     version_added: "2.8"
+  apply:
+    description:
+    - Whether to use 'apply' method to update resources.
+    - "'apply' remembers previous invocations of k8s apply through an annotation"
+    - "'apply' attempts to ensure a resource is exactly as specified - it's closer to a replace than a patch."
```

Comment:
I don't think I'd say this, it's more like a really smart patch that can automatically compute deletions. It's still going to be sending a PATCH and not a PUT so I'd just worry someone gets the wrong idea.

Reply:
Thanks, I'll rework the wording